conf/uppmax.config

```diff
@@ -9,13 +9,16 @@ vim: syntax=groovy
  * profile in nextflow.config
  */
 
+
+params.rlocation = "$HOME/R/nxtflow_libs/"
```

Comment:
We already have a `params` section lower down, better to put this under the `clusterOptions = false` line.